docs/appendices/release-notes/unreleased.rst

```diff
@@ -56,6 +56,8 @@ None
 Changes
 =======
 
+- Implemented ``UNION DISTINCT``(``UNION``).
```

Comment:
Could maybe be a bit more verbose - like `Added the UNION DISTINCT statement to support retrieving unique rows from multiple relations without using sub-queries with extra GROUP BY clause`

And it could link via `:ref:` directly to the dos.